dbt/models/proximity/proximity.dist_pin_to_new_construction.sql

```diff
@@ -0,0 +1,38 @@
+{{
+    config(
+        materialized='table',
+        partitioned_by=['year'],
+        bucketed_by=['pin10'],
+        bucket_count=1
+    )
+}}
+
+WITH new_construction AS (  -- noqa: ST03
+    SELECT
+        parcel.pin10,
+        ST_ASBINARY(ST_POINT(parcel.x_3435, parcel.y_3435)) AS geometry_3435,
+        parcel.year,
+        vw_card_res_char.char_yrblt
+    FROM spatial.parcel AS parcel
+    INNER JOIN default.vw_card_res_char AS vw_card_res_char
+        ON parcel.pin10 = vw_card_res_char.pin10
+        AND parcel.year = vw_card_res_char.year
+        AND CAST(parcel.year AS INT)
+        <= CAST(vw_card_res_char.char_yrblt AS INT) + 2
```

Comment:
**suggestion:** Let's actually bump this up to 3 years, 2 seems a bit short to me.